docs/advanced-features/tabulator.md

```diff
@@ -0,0 +1,100 @@
+Here is the revised version with the API call information moved into the **Overview** section, and the **Configuration** section focused solely on the YAML file:
+
+---
+
+# Tabulator Configuration for Quilt
+
+> This feature requires Quilt stack version 1.55.0 or higher
+
+## Overview
+
+Tabulator allows Quilt admins to configure tables for querying tabular data
+objects across multiple packages using AWS Athena. Admins can define schemas
+and data sources for CSV, TSV, Parquet, and other formats, enabling users to
+run SQL queries directly on Quilt packages.
+
+The configuration can be written in YAML and managed using the
+`quilt3.admin.tabulator_config.set()` function or via the Quilt Admin UI.
```

Comment:
that should become link when https://github.com/quiltdata/quilt/pull/4136 is merged

Reply:
What's the (relative) URL I should use?

Reply:
../api-reference/Admin.md ?